### PR TITLE
Add Sanitized HTML and Markdown column type

### PR DESCRIPTION
**Pull request by vitPinchuk**
Resolves: #150

### CODE REVIEW

**Review by mikhail-vl — COMMENTED** [review]
@asimonok Can we rename `Auto` to `Text`. Or there are more to it?

File: provisioning/dashboards/panels.json [hunk 1]
```diff
@@ -1136,6 +1189,7 @@
           "export": true
         }
       },
+      "pluginVersion": "1.6.0",
```
Comment (asimonok) [inline provisioning/dashboards/panels.json:1192]: pluginVersion shouldn't be saved in provisioned dashboards to make migration always working

File: src/components/Table/components/CellRenderer/LayoutCellRenderer.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+   */
+  const md = new MarkdownIt({
+    html: true,
+  });
+
+  /**
+   * Context
+   */
+
+  const { replaceVariables } = tablePanelContext.useContext();
```
Comment (asimonok) [inline src/components/Table/components/CellRenderer/LayoutCellRenderer.tsx:36]: I would add current row as a scoped var

**Review by asimonok — CHANGES_REQUESTED** [review]

**Review by asimonok — APPROVED** [review]
Looks good!

@mikhail-vl ready for review

**Review by mikhail-vl — APPROVED** [review]
Looks great!

**Pull request merged**